src/components/sections/achievement.section.tsx

```diff
@@ -1,15 +1,14 @@
 import { Box, Flex, H2, H3, Paragraph } from '@cogito-study/alea';
 import { graphql, useStaticQuery } from 'gatsby';
 import React, { FunctionComponent } from 'react';
-import { AchievementQuery } from '../../generated/graphql-types';
 import { Container } from '../container';
 
 interface AchievementSectionProps {
   title: string;
 }
 
 export const AchievementSection: FunctionComponent<AchievementSectionProps> = ({ title }) => {
-  const data = useStaticQuery<AchievementQuery>(graphql`
+  const data = useStaticQuery<any>(graphql`
```

Comment:
faszan kiszedted, csak akkor mar ide nem is kell generikus tipuscontraint
```suggestion
  const data = useStaticQuery(graphql`
```